.travis.yml

```diff
@@ -9,22 +9,30 @@ addons:
       - libqt4-opengl-dev
       - g++-4.8
       - python3
+      - python-pip
+      - python3-pip
 compiler:
   - clang
 env:
   - py=python2
   - py=python3
-install: 
+install:
   - export NUMBER_OF_PROCESSORS=4
-  - export PATH=`pwd`/release/bin:`pwd`/scripts:${PATH}
+  - export PATH=`pwd`/bin:${PATH}
   - export EIGEN_CFLAGS=-I`pwd`/../eigen
   - (cd ..; hg clone https://bitbucket.org/eigen/eigen/; cd eigen; hg update 3.3)
+  - pip install recommonmark sphinx sphinx-rtd-theme
+  - pip3 install recommonmark sphinx sphinx-rtd-theme
 script:
+  - ${py} -m sphinx -n -N -W -w sphinx.log -c docs/ docs/ tmp/
   - ./check_memalign && $py ./configure -assert && $py ./build -nowarnings && ./run_tests
+  - ./docs/generate_user_docs.sh && ! ( git diff -- . ':(exclude)docs/reference/commands/mrview.rst' ':(exclude)docs/reference/commands/shview.rst' | tee gitdiff.log | grep '^-\|^+' | grep -q -v 'commands_list.rst\|mrview\|shview' )
```

Comment:
Not sure I follow... But in any event, I'm trying to get the GUI commands to work as expected with these options even without a display, which would obviate the need for this.